tests/UI/campaigns/functional/FO/03_userAccount/04_orderList.js

```diff
@@ -0,0 +1,81 @@
+require('module-alias/register');
+
+const {expect} = require('chai');
+
+const helper = require('@utils/helpers');
+
+// Import data
+const {DefaultAccount} = require('@data/demo/customer');
+
+// Importing pages
+const foHomePage = require('@pages/FO/home');
+const foLoginPage = require('@pages/FO/login');
+const foMyAccountPage = require('@pages/FO/myAccount');
+const foOrderHistoryPage = require('@pages/FO/myAccount/orderHistory');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_FO_userAccount_orderList';
+
+let browserContext;
+let page;
+
+const numberOfDefaultOrders = 5;
+
+/*
+Sign in FO with default account
+Go to orders history page
+Check that number of orders is above 5 (default orders)
+ */
+
+describe('Check number of orders in order history page', async () => {
+  // before and after functions
+  before(async function () {
+    browserContext = await helper.createBrowserContext(this.browser);
+    page = await helper.newTab(browserContext);
+  });
+
+  after(async () => {
+    await helper.closeBrowserContext(browserContext);
+  });
+  it('should go to FO home page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToFoToCreateAccount', baseContext);
+
+    await foHomePage.goToFo(page);
+    const isHomePage = await foHomePage.isHomePage(page);
+    await expect(isHomePage).to.be.true;
+  });
+
+  it('should go to login page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToLoginFoPage', baseContext);
+
+    await foHomePage.goToLoginPage(page);
+
+    const pageHeaderTitle = await foLoginPage.getPageTitle(page);
+    await expect(pageHeaderTitle).to.equal(foLoginPage.pageTitle);
+  });
+
+  it('Should sign in FO', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'signInFo', baseContext);
+
+    await foLoginPage.customerLogin(page, DefaultAccount);
+    const isCustomerConnected = await foMyAccountPage.isCustomerConnected(page);
+    await expect(isCustomerConnected, 'Customer is not connected').to.be.true;
+  });
+
+  it('should go to order history page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToOrderHistoryPage', baseContext);
+
+    await foMyAccountPage.goToHistoryAndDetailsPage(page);
+    const pageHeaderTitle = await foOrderHistoryPage.getPageTitle(page);
+    await expect(pageHeaderTitle).to.equal(foOrderHistoryPage.pageTitle);
+  });
+
+  it('should go to order history page', async function () {
```

Comment:
```suggestion
  it('should see order listing', async function () {
```

Reply:
Not sure about the string, but you forgot to change it anyway

Reply:
It's the right string, no need to change it